pkg/controller/kibana/config_settings.go

```diff
@@ -43,6 +43,8 @@ const (
 	XpackMonitoringUIContainerElasticsearchEnabled = "xpack.monitoring.ui.container.elasticsearch.enabled"
 	XpackLicenseManagementUIEnabled                = "xpack.license_management.ui.enabled" // >= 7.6
 	XpackSecurityEncryptionKey                     = "xpack.security.encryptionKey"
+	XpackReportingEncryptionKey                    = "xpack.reporting.encryptionKey"
+	XpackEncryptedSavedObjectsEncryptionKey        = "xpack.encryptedSavedObjects.encryptionKey"
```

Comment:
I believe this setting is not available in all version of Kibana we support. I am trying to figure out when it was added but it is definitely not in 6.8

Reply:
Yep and it errors out too :/

>{"type":"log","@timestamp":"2020-06-18T13:45:18Z","tags":["fatal","root"],"pid":1,"message":"{ Error: \"xpack.encryptedSavedObjects.encryptionKey\" setting was not applied. Check for spelling errors and ensure that expected plugins are installed.\n    at KbnServer.exports.default (/usr/share/kibana/src/server/config/complete.js:88:17) code: 'InvalidConfig', processExitCode: 64 }"}

> FATAL  Error: "xpack.encryptedSavedObjects.encryptionKey" setting was not applied. Check for spelling errors and ensure that expected plugins are installed.